app/models/pipeline_run.rb

```diff
@@ -402,7 +402,7 @@ def multihit?
   end
 
   def assembly?
-    after(pipeline_version || fetch_pipeline_version, "1.11")
+    after(pipeline_version || fetch_pipeline_version, "1000.1000")
```

Comment:
which means no assembly?

Reply:
yes. We decided to hold off on pushing the assembly pipeline (or other new features) until after the refactor.

Reply:
put some documentation on it to indicate the intention. This code looks cryptic by itself.

Reply:
ok!